### PR TITLE
Send payload back for zygote cell

### PR DESCRIPTION
**Pull request by dturn**
r: @dalehamel 

Burnin cell requires the payload to populate the params

related to https://github.com/Shopify/zygote/pull/70

### CODE REVIEW

File: init/onboot_script [hunk 1]
```diff
@@ -29,9 +29,10 @@ if [ -n "$(grep onboot_script_url= /proc/cmdline)" ];then
 
   # Read the script URL from /proc/cmdline
   url_param=$(grep -o 'onboot_script_url=\S*' /proc/cmdline)
+  payload=$(grep -o 'payload=\S*' /proc/cmdline)
   url=${url_param#onboot_script_url=}
 
-  curl $url > /onboot_script
+  curl "$url?$payload" > /onboot_script
```
Comment (dalehamel) [inline init/onboot_script]: best to put { } around the variable, IE

```
"${url}?${payload}"
```

**Comment by dalehamel** [comment]
looks good

**Comment by dalehamel** [comment]
acutally, let's make sure payload is not empty, don't send back an empty payload.

File: init/onboot_script [hunk 2]
```diff
@@ -31,7 +31,12 @@ if [ -n "$(grep onboot_script_url= /proc/cmdline)" ];then
   url_param=$(grep -o 'onboot_script_url=\S*' /proc/cmdline)
   url=${url_param#onboot_script_url=}
 
-  curl $url > /onboot_script
+  payload=$(grep -o 'payload=\S*' /proc/cmdline)
+  if [ -z  ${payload#payload=} ];then
+    payload=""
+  fi
```
Comment (dturn) [inline init/onboot_script:37]: @dalehamel This look like a reasonable way to?: 

> make sure payload is not empty, don't send back an empty payload.

File: init/onboot_script [hunk 3]
```diff
@@ -31,7 +31,12 @@ if [ -n "$(grep onboot_script_url= /proc/cmdline)" ];then
   url_param=$(grep -o 'onboot_script_url=\S*' /proc/cmdline)
   url=${url_param#onboot_script_url=}
 
-  curl $url > /onboot_script
+  payload=$(grep -o 'payload=\S*' /proc/cmdline)
+  if [ -z  ${payload#payload=} ];then
```
Comment (dalehamel) [inline init/onboot_script]: i think you'll want to quote the ${} part of this

**Comment by dwradcliffe** [comment]
For some reason, this change is breaking bootstrap on _(some?)_ servers. Seems to be a syntax issue.

**Comment by dturn** [comment]
Fixed in https://github.com/Shopify/livesys/pull/2

**Pull request merged**